comp/netflow/goflowlib/convert.go

```diff
@@ -74,6 +75,11 @@ func applyAdditionalFields(flow *common.Flow, additionalFields common.Additional
 		if applied {
 			// We replaced a field of common.Flow with an additional field, no need to keep it in the map
 			delete(additionalFields, destination)
+		} else {
+			if field, ok := fieldValue.([]byte); ok {
```

Comment:
Are the changes in this PR united tested?

Reply:
I don't see test changes beside: https://github.com/DataDog/datadog-agent/pull/20872/files#diff-554e8957a1e4497f84e183b5516fc30fe2232abd39d32535c91b66595c15031fR95